Dangerfile.swift

```diff
@@ -0,0 +1,144 @@
+    warn("Consider adding supporting documentation to this change. Documentation can be found in the `docs` directory.")
+}
+
+// Run danger-prose to lint Chinese docs
+func isCnDocsMarkdown(_ file: String) -> Bool {
+    return file.fileType == .markdown && file.contains("docs_CN")
+}
+let addedAndModifiedCnDocsMarkdown = danger.git.createdFiles.filter(isCnDocsMarkdown) + danger.git.modifiedFiles.filter(isCnDocsMarkdown)
+if #available(OSX 10.12, *),
+    !addedAndModifiedCnDocsMarkdown.isEmpty {
+    Proselint.performSpellCheck(files: addedAndModifiedCnDocsMarkdown, excludedRules: ["misc.scare_quotes", "typography.symbols"])
+}
+
+// Run danger-prose to lint and check spelling English docs
+func isEnDocsMarkdown(_ file: String) -> Bool {
+    return file.fileType == .markdown && file.contains("docs/")
+}
+let addedAndModifiedEnDocsMarkdown = danger.git.createdFiles.filter(isEnDocsMarkdown) + danger.git.modifiedFiles.filter(isEnDocsMarkdown)
+if #available(OSX 10.12, *),
+    !addedAndModifiedEnDocsMarkdown.isEmpty {
+    Proselint.performSpellCheck(files: addedAndModifiedEnDocsMarkdown)
+
+    let ignoredWords = ["Auth", "auth", "Moya", "enum", "enums", "OAuth", "Artsy's", "Heimdallr.swift", "SwiftyJSONMapper", "ObjectMapper", "Argo", "ModelMapper", "ReactiveSwift", "RxSwift", "multipart", "JSONEncoder", "Alamofire", "CocoaPods", "URLSession", "plugin", "plugins", "stubClosure", "requestClosure", "endpointClosure", "Unsplash", "ReactorKit", "Dribbble", "EVReflection", "Unbox"]
+    Mdspell.performSpellCheck(files: addedAndModifiedEnDocsMarkdown, ignoredWords: ignoredWords, language: "en-us")
+}
+
+// Warning message for not updated package manifest(s)
+let manifests = [
+    "Moya.podspec",
+    "Cartfile",
+    "Cartfile.resolved",
+    "Package.swift",
+    "Package.resolved"
+]
+let updatedManifests = manifests.filter { manifest in danger.git.modifiedFiles.contains { $0.name == manifest } }
+if !updatedManifests.isEmpty && updatedManifests.count != manifests.count {
+    let notUpdatedManifests = manifests.filter { !updatedManifests.contains($0) }
+    let updatedArticle = updatedManifests.count == 1 ? "The " : ""
+    let updatedVerb = updatedManifests.count == 1 ? "was" : "were"
+    let notUpdatedArticle = notUpdatedManifests.count == 1 ? "the " : ""
+
+    warn("\(updatedArticle)\(updatedManifests.joined(separator: ", ")) \(updatedVerb) updated, " +
+        "but there were no changes in \(notUpdatedArticle)\(notUpdatedManifests.joined(separator: ", ")).\n" +
+        "Did you forget to update them?")
+}
+
+// Warn when library files has been updated but not tests.
+let testsUpdated = danger.git.modifiedFiles.first { $0.hasPrefix("Tests") } != nil
+if sourceChanges && !testsUpdated {
+    warn("The library files were changed, but the tests remained unmodified. Consider updating or adding to the tests to match the library changes.")
+}
+
+// Run Swiftlint
+SwiftLint.lint(inline: true)
```

Comment:
I remember we had a problem with SwiftLint plugin to Danger Swift and we had to use it two times, once to lint `Sources` and once to lint `Tests` (see https://github.com/ashfurrow/danger-swiftlint/issues/4#issuecomment-364797573) - do you know if this is fixed already?

Reply:
Also, can we add `Dangerfile.swift` to SwiftLint? 😄

Reply:
We can and we should :P

Reply:
Danger Swiftlint supports the config file then I would try to create one that includes `Sources` and `Tests` and see what happens, we can still go with the double lint if needed

Reply:
Swiftlint is not using`.swiftlint.yml` config file that contains
```yaml
included:
  - Sources
  - Tests
  - Examples
  - Dangerfile.swift
```
Let's see how it goes, if there is any problem I will change it :)